editor.planx.uk/src/@planx/components/shared/Schema/InputFields/TextFieldInput.tsx

```diff
@@ -12,6 +14,7 @@ export const TextFieldInput: React.FC<Props<TextField>> = (props) => {
   const { data, formik } = props;
   const { id, errorMessage } = fieldProps;
 
+  const characterCountLimit = data.type && isLongTextType(data.type);
```

Comment:
For me, it made sense to keep this here so when you look into the TextFieldInput, you understand why things show and why things don't without ever having to go into the CharacterCount component. To understand the TextFieldInput, you don't need to go to another file, in theory